docs/data/material/integrations/interoperability/interoperability.md

```diff
@@ -586,7 +586,7 @@ For Tailwind CSS v4, please refer to the [v4 integration guide](/material-ui/in
 
 <!-- #target-branch-reference -->
 
-If you are used to Tailwind CSS and want to use it together with the Material UI components, you can start by cloning the [Tailwind CSS](https://github.com/mui/material-ui/tree/master/examples/material-ui-cra-tailwind-ts) example project.
+To use Tailwind CSS with Material UI components, you can start by cloning the [example project](https://github.com/mui/material-ui/tree/master/examples/material-ui-vite-tailwind-ts) built with Vite and TypeScript.
```

Comment:
This is under the `Tailwind CSS v3` section, however, the example of `material-ui-vite-tailwind-ts` uses Tailwind CSS v4. Update the example to use version 3 instead.